velox/exec/ExchangeQueue.h

```diff
@@ -119,8 +131,20 @@ class ExchangeQueue {
   ///
   /// The data may be compressed, in which case 'maxBytes' applies to compressed
   /// size.
+  std::vector<std::unique_ptr<SerializedPage>> dequeueLocked(
+      int consumerId,
+      uint32_t maxBytes,
+      bool* atEnd,
+      ContinueFuture* future,
+      std::vector<ContinuePromise>& promises);
```

Comment:
```
s/promises/staledPromises/
```
Maybe put a comment? And at the caller, we expect it at most has one promise?

Reply:
Actually yeah, let me use a pointer instead of a list. It should never be more than one